### DISCUSSION TITLE
Average run time?

### DISCUSSION THREAD

**Discussion by Tharic99** [meisnate12/Plex-Meta-Manager-Configs#10] [General]
I'm just curious what others are seeing for fun times on this? I've got 446 collections built in my yaml file currently and I've noticed that when the script runs, it's taking around 7 or 8 hours to complete.

**Comment by meisnate12** [comment]
I have 544 movie collections and it takes me about 4:45 to complete just for movies I don't have TV or Anime currently running

**Reply by Tharic99** [reply]
NIce. I realized why mine was taking so long...

```
radarr:
  add: true
```
Cut my time in half when I removed that line. Even though I don't have any of the add_to_arr in the collections, that was still turned on and it looked like it was trying to add to Radarr on every item.

**Reply by meisnate12** [reply]
yea i dont have many that add to radarr
